CHANGELOG.md

```diff
@@ -153,6 +153,7 @@ CHANGELOG
 - `intelmq.bots.experts.threshold.expert`: Correctly use the standard parameter `redis_cache_ttl` instead of the previously used parameter `timeout` (PR#2155 by Karl-Johan Karlsson).
 - `intelmq.bots.experts.jinja2.expert`: Lift restriction on requirement jinja2 < 3 (PR#2158 by Sebastian Wagner).
 - `intelmq.bots.experts.asn_lookup.expert`, `intelmq.bots.experts.domain_suffix.expert`, `intelmq.bots.experts.maxmind_geoip.expert`, `intelmq.bots.experts.recordedfuture_iprisk.expert`, `intelmq.bots.experts.tor_nodes.expert`: New parameter `autoupdate_cached_database` to disable automatic updates (downloads) of cached databases (PR#2180 by Sebastian Wagner).
+- `intelmq.bots.experts.url.expert`: New bot for extracting additional information from `source.url` and/or `destination.url` (PR#2315 by Filip Pokorný).
```

Comment:
I am missing an entry in NEWS.md to describe the new bot: add a sentence what it is, and one why it was needed and why it should replace the old one. Otherwise, I think it's good! Thank you very much for your contribution @gethvi

Reply:
Yes, I just talked with him. I will add a line in NEWS.md to announce that the new bot will replace the old url2fqdn (that was my idea). All good :) I'll do it.